app/src/lib/stats/stats-store.ts

```diff
@@ -84,12 +87,18 @@ type DailyStats = ICalculatedStats & ILaunchStats & IDailyMeasures
 /** The store for the app's stats. */
 export class StatsStore {
   private readonly db: StatsDatabase
+  private readonly uiActivityMonitor?: IUiActivityMonitor
+  private uiActivityMonitorSubscription: Disposable | null = null
 
   /** Has the user opted out of stats reporting? */
   private optOut: boolean
 
-  public constructor(db: StatsDatabase) {
+  public constructor(
+    db: StatsDatabase,
+    uiActivityMonitor?: IUiActivityMonitor
```

Comment:
Is there ever a case where we won't be passing this type to the store?

Reply:
I made it optional to not have to bother with creating an supplying a fake activity monitor in tests such as `app-test`. I don't feel strongly about it